tests/filesystem/test_bash.py

```diff
@@ -9,22 +8,23 @@
 from datetime import datetime, timedelta
 from typing import Tuple, List
 
-TEMP_DIR = Path(tempfile.gettempdir())
-
 
-def _make_old_files(directory_path: str, num_days_old: int) -> Tuple[List, List]:
+def _make_old_files(
+    directory_path: str, num_days_old: int, temporary_directory: Path
```

Comment:
what's directory_path?

Reply:
This is to allow for creation of more specific paths within the temporary directory. I'll add it to the docstring for this helper function.